src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Ball {
+    enum Colors {
```

Comment:
Move this enum out of class Ball to a separate file
And fix name Colors -> **Color**